README.md

```diff
@@ -19,6 +19,8 @@ Require the [go programming language](https://golang.org/) (version >= 1.13)
 ```
 git clone https://github.com/edouardparis/lntop.git
 cd lntop && go install ./...
+go build -o lntop cmd/lntop/main.go
```

Comment:
Can you separate it from the go install command in a new code snippet like this:

--

If you do not want to have the binary inside your $GOPATH, you can create it and run it inside the repository:
```
go build -o lntop cmd/lntop/main.go
./lntop
```